Loop/Settings/ExcludeListSettingsView.swift

```diff
@@ -126,8 +126,8 @@ struct ExcludeListSettingsView: View {
                     .padding(-10)
                 } header: {
                     VStack(alignment: .leading) {
-                        Text("Excluded Applications")
-                        Text("Applications in the exclude list are ignored by Loop.")
+                        Text("Excluded applications")
```

Comment:
Titles/headers should be in title case, meaning this would be "Excluded Applications"

Reply:
Updated.